packages/medusa-js/src/resources/admin/customers.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  AdminCustomersListRes,
+  AdminCustomersRes,
+  AdminGetCustomersParams,
+  AdminPostCustomersReq,
+} from "@medusajs/medusa"
+import { ResponsePromise } from "../.."
+import BaseResource from "../base"
+
+class AdminCustomersResource extends BaseResource {
+  /**
+   * Creates a customer
+   * @param {AdminPostCustomersReq} payload information of customer
```

Comment:
Are the types here necessary?
```suggestion
   * @param payload - information of customer
```

Reply:
Have deleted param types and the return type entirely @srindom